scripts/seed-database/write/main.sql

```diff
@@ -1,6 +1,4 @@
-BEGIN;
 \include write/users.sql
 \include write/teams.sql
 \include write/flows.sql
-\include write/flow_document_templates.sql
-COMMIT;
+\include write/flow_document_templates.sql
```

Comment:
should `published_flows` be included on this list too ? i think it's a bit more clear to have a reference of all synced tables in same list and would mean we can remove this line (i think) https://github.com/theopensystemslab/planx-new/pull/2177/files#diff-ac583dce0ccc6237bde4c2e74af81ce849d30a30fc84beba82b143b6a2dc5617R43